### DISCUSSION TITLE
0.3.1

### DISCUSSION THREAD

**Discussion by Spidy123222** [SideStore/SideStore#283] [Announcements]
This is **unreleased software** and we do not guarantee to it to work for everyone. If you have any issues or questions join the jitstreamer discord. We will do a 1.0.0 when we have a stable version of sidestore and have a update release accordingly.

**Make sure sidestore.conf is imported to WireGuard and enabled**

Happy March! We have been busy with doing quality of life changes and getting stuff up and running like deactivating and getting bugs fixed that you guys found in nightly. We don’t currently have a fix for expiring early issue but is in the works like anisette issues. This should allow basic support for all functions.

For people that want to use MDC no3applimit (MacDirtyCow). It’s currently not ready as it includes the patch not needing any external apps. So for any brave people that really want if you will have to look at this pull request and use the ipa downloads from GitHub actions  https://github.com/SideStore/SideStore/pull/259 . We don’t recommend relying on pull requests as because a commit could break the app so be warned, this is the same for nightly but with nightly has latest develop branch changes.

## What's Changed
- Fix beta apps not refreshing
- Fix beta apps on the my apps view 
- Fix deactivating/activating apps
- Merge all of Riley's new error handling stuff
- add easy anisette server selector iOS Settings app
- added toggle for custom anisette server
- Change the default anisette server
- Add a bit more logging throughout the signing process to help us pin down issues
- Allow for .mobiledevicepairing and .plist files to be selected, try and pin down why users can't select either
- Hopefully fix AltWidget issues
- Add analytics to help developers understand current demographic of users (and hopefully pin down version specific bugs)
- Icon change (thanks LittRitt!)
- Sidestore directory accessible in files
- Reset pairing file button
- Advanced settings button
- Browse icon change
- Credit adjustment
- Background color fixes
- Added more permission types for sources
- Join Patreon button crash fix
- Turned off background notification refreshes
- Sidestore Logger 
- Minimuxer logger
- Additional logging for minimuxer for sidestore for troubleshooting
- Replace Placeholder video for pairing instructions.
- Fix BundleId issues on parts of sidestore
- Fix trusted sources if one app has problems that it shouldn’t take down the full list.

## New Contributors!

Also big thanks to our new contributors for helping with SideStore!
* @fabianthdev
* @LitRitt
* @f1shy-dev
* @naturecodevoid
* @SoY0ung
* @oq-x

**Full Commit Changelog**: https://github.com/SideStore/SideStore/compare/0.1.1...0.3.1

<hr /><em>This discussion was created from the release <a href='https://github.com/SideStore/SideStore/releases/tag/0.3.1'>0.3.1</a>.</em>

**Comment by nextDNSconfig** [comment]
So the after 7 days the SideStore app won't refresh. And you still need a computer to make it work

**Comment by Spidy123222** [comment]
> So the after 7 days the SideStore app won't refresh. And you still need a computer to make it work

You refresh before 7 days otherwise the app would be expired. You don't need a computer to refresh sidestore or adding apps. sideserver and altserver were only used for installing. You just need WiFi to use sidestore using the ondevice vpn via WireGuard.

**Reply by nextDNSconfig** [reply]
I get SideStore is no longer available, but I refreshed it a day before ![image](https://user-images.githubusercontent.com/86489640/222361511-35deacfe-c512-41e9-8835-5ab096862fa6.png)

**Comment by lrs95** [comment]
After refreshing my app, the time in the View App IDs still hasn't changed.

**Comment by amsam0** [comment]
> After refreshing my app, the time in the View App IDs still hasn't changed.

Please read the text in View App IDs.

**Reply by nextDNSconfig** [reply]
![image](https://user-images.githubusercontent.com/86489640/223079588-9caabea8-f0bd-4a58-b0e8-a2a764fef4b5.png)

![image](https://user-images.githubusercontent.com/86489640/223079984-3bc42cb6-7b7c-4429-8f26-15610fd35e97.png)

Look: in does not refresh the app Id

**Reply by amsam0** [reply]
Read the text in view app IDs. It tells you all you need to know.
> SideStore will automatically renew app IDs once they expire.

**Reply by likereadboy** [reply]
when app really expired, i can’t open it either, no automatically renew actions seems was done.especially the sidestore app itself.

**Reply by amsam0** [reply]
If SideStore expires you have to reinstall via computer. It can’t do anything if it expires. App IDs are NOT the same as provisioning profiles, which when they expire cause SideStore to expire.

**Reply by likereadboy** [reply]
can i keep sidestore not expired just use itself or not?

> If SideStore expires you have to reinstall via computer. It can’t do anything if it expires. App IDs are NOT the same as provisioning profiles, which when they expire cause SideStore to expire.

**Reply by amsam0** [reply]
Yes, if you refresh it before it expires it will stay. There is a bug where SideStore will expire early, but it's one of our top priority bugs and many people are looking into it.

**Comment by Karl0226** [comment]
I have a problem, I try the latest minimuxer, it can run on iPhoneX, but it can't run on iPhone13, the received signal is always ListDevices
